lib/eslint/eslint.js

```diff
@@ -950,10 +953,13 @@ class ESLint {
 
                         return result;
                     }).catch(error => {
-                        controller.abort(error);
-                        throw error;
-                    });
 
+                        if (error.name !== "AbortError" && !retryCodes.has(error.code)) {
```

Comment:
Normally it shouldn't be necessary to exclude `AbortError`s explicitly. Those errors will only pop up after the `AbortController` has been signaled abort, and at that point, calling the `abort` method one more time will have no effect anyway.

Did you add the `error.name` check because of some failing tests?

Reply:
Good call out. I was having some trouble with aborted requests when I was writing the tests, but I just realized it's because the `catch` is in the wrong place. Moving it means I can remove all of this check.